advanced_tutorials/llm_pdfs/requirements.txt

```diff
@@ -0,0 +1,22 @@
+google-api-python-client==2.114.0
+httplib2==0.22.0
+oauth2client==4.1.3
+pydrive==1.3.1
+PyPDF2==3.0.1
+pandas==2.1.4
+sentence-transformers==2.2.2
+accelerate==0.26.1
+peft==0.7.1
+bitsandbytes==0.40.2
+transformers==4.36.2
+flask-sqlalchemy==3.1.1
+trl==0.7.9
+langchain==0.1.1
+pyopenssl==23.3.0
+FlagEmbedding
+streamlit==1.30.0
+openai==1.9.0
+getpass4==0.0.14.1
+json_repair==0.6.1
+protobuf==3.20.0
+git+https://github.com/logicalclocks/feature-store-api@master#egg=hsfs[python]&subdirectory=python
```

Comment:
Can you change this to be just
hopsworks
We don't want to install master

Reply:
Done ✅